test/extensions/filters/network/postgres_proxy/postgres_integration_test.cc

```diff
@@ -172,7 +172,7 @@ TEST_P(SSLWrongConfigPostgresIntegrationTest, TerminateSSLNoStartTlsTransportSoc
   // Message will be processed by Postgres filter which
   // is configured to accept SSL termination and confirm it
   // by returning single byte 'S'.
-  ASSERT_TRUE(tcp_client->write(data.toString()));
+  ASSERT_TRUE(tcp_client->write(data.toString(), false, false));
```

Comment:
Looks good. Can we put a comment on each boolean flag? Thanks!

Reply:
I added a comment but I am not sure if this is what you mean. Let me know if I should elaborate more

Reply:
This makes sense, as after sending "S", the postgres filter will close TCP connection, because _starttls_ transport socket is missing from config.